### PR TITLE
Feat/share posts

### PR DESCRIPTION
**Pull request by guilhermeicarofr**
What's new:
 - Now when you share or unshare a repost the hashtags involved in it influence on hashtags trending

### CODE REVIEW

File: src/controllers/hashtags.controllers.js [hunk 1]
```diff
@@ -44,9 +44,10 @@ async function getHashtagsByName(req, res) {
 				};
 			})
 		);
-
+		console.log(completePosts)
```
Comment (MacenaAndre) [inline src/controllers/hashtags.controllers.js]: Remove console.log

File: src/controllers/hashtags.controllers.js [hunk 2]
```diff
@@ -44,9 +44,10 @@ async function getHashtagsByName(req, res) {
...
 		res.status(200).send(completePosts);
 	} catch (error) {
+		console.log(error)
```
Comment (MacenaAndre) [inline src/controllers/hashtags.controllers.js]: Remove console.log

**Review by MacenaAndre — APPROVED** [review]
Excellent!!

**Review by tissyane — APPROVED** [review]
Well done!!

**Pull request merged**